<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QFrame" name="infoComplementaireOrange">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type incident</string>
     </property>
    </widget>
    <widget class="QComboBox" name="type_incident">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Interruption</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dégradation</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="qualite">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>HD et SD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SD</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Qualité</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="zap">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zap</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="declenchement_partenaire">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Déclenchement fournisseur</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="numeroTT">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numéro du TT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TDR</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chaine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chaine</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TDR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GCF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Autre</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Problème</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="probleme">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Commentaire</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commentaire">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>numeroTT</tabstop>
  <tabstop>TDR</tabstop>
  <tabstop>chaine</tabstop>
  <tabstop>probleme</tabstop>
  <tabstop>commentaire</tabstop>
  <tabstop>zap</tabstop>
  <tabstop>type_incident</tabstop>
  <tabstop>qualite</tabstop>
  <tabstop>declenchement_partenaire</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
